### PR TITLE
get errors only from a specific path

### PR DESCRIPTION
**Pull request by Thamaraiselvam**
`$client->setStrictErrorCapturePath($path);`

unit tested works fine.

### CODE REVIEW

**Comment by Thamaraiselvam** [comment]
maintainers, can you check this pull request?

File: lib/Raven/Client.php [hunk 1]
```diff
@@ -414,6 +426,12 @@ public function captureMessage($message, $params=array(), $data=array(),
      */
     public function captureException($exception, $data=null, $logger=null, $vars=null)
     {
+        //avoid errors other than specific path
+        $exception_file = $exception->getFile();
+        if (isset($this->strict_error_capture_path) && !empty($this->strict_error_capture_path) && stripos($exception_file, $this->strict_error_capture_path) === false) {
```
Comment (stayallive) [inline lib/Raven/Client.php]: You can leave out the `isset` check and leave only the `empty`.

See: http://stackoverflow.com/a/4559976/1580028

**Comment by stayallive** [comment]
Did you perhaps forget to also include the unit tests?

File: lib/Raven/Client.php (hunk 1 above)
Comment (Thamaraiselvam) [inline lib/Raven/Client.php]: Thanks for the review. Removed `isset`

**Comment by Thamaraiselvam** [comment]
I have tested it and using it in my application.

**Comment by Thamaraiselvam** [comment]
Okay. I did very basic tests.
I set my plugin path 
`$sentry_client->setStrictErrorCapturePath($my_plugin_dir);`

I get the errors only on my plugin path. even other plugins or Wordpress throws the errors it just ignores them.

**Comment by nokitakaze** [comment]
This can not be approved without code coverage (unit tests)

**Comment by ste93cry** [comment]
Wouldn't be better to use the `send_callback` option to customize whether the event originated from a specific source (directory or file) should be sent?

**Comment by nokitakaze** [comment]
Already done in 2.0

**Comment by stayallive** [comment]
Thanks for the contribution. This has indeed been implemented in 2.0, if you would still like it in the 1.x version please reopen the PR with the changes/tests requested above.

**Comment by nokitakaze** [comment]
There is no need to add new behaviour to old version since it (version 1.0) will be stalled from 2018 March.

**Pull request closed without merging**